enterprise/ecommerce_api.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to get details from the ecommerce API.
+"""
+from __future__ import absolute_import, unicode_literals
+
+try:
+    from openedx.core.djangoapps.commerce.utils import ecommerce_api_client, is_commerce_service_configured
+except ImportError:
+    def is_commerce_service_configured():
+        """
+        When we call this name, return False, since we couldn't import.
+        """
+        return False
+
+    ecommerce_api_client = None
+
+
+class EcommerceApiClient(object):
+    """
+    Object builds an API client to make calls to the Ecommerce API.
```

Comment:
@haikuginger Nit: Standardize spelling: `ecommerce API` vs. `Ecommerce API`. I'd slightly prefer the former.